README.md

```diff
@@ -1,4 +1,46 @@
 # machine_learning_notes
+
 As part of ***Machine Learning Study Group*** from recworks meet-a-mentor community
 
-work on kaggle ML project: https://www.kaggle.com/c/house-prices-advanced-regression-techniques
+work on kaggle ML project.
+
+We use Jupyter Notebook, Python and some libraries (Pandas, NumPy, Matplotlib and Scikit-learn) to solve ML problems on public data sets. We have started with the following data set from Kaggle:
+
+https://www.kaggle.com/c/house-prices-advanced-regression-techniques.
+
+This project is the result of the joined effort of a study group of enthusiastic people from different professional background. Anyone is welcome to join and to contribute with the discussion. 
+
+The best way to join to the project is by assisting to on our regular session.  Also, feel free to fork and check what we have done in this project. We are very happy to receive any comment and suggestions to improve.
+
+
+There are two notebooks contained in the repository:
+
+## 1) machine_learning_notes.ipynb:
+
+This notebook contains:
+
+* 1 - Define the problem
+* [Load data and displaying info](#load-data)
+* 2 - Prepare Data
+    * [Identify features](#identify)
+        * Separate numerical from categorical features
+        * Separate nominal and ordinal (from categorical features)
+    * [Clean data](#clean)
+        * Remove numerical features with missing values
+        * Remove categorical features with missing values
+        * drop outliers in numerical values # WIP
+    * [transform](#transform) # TODO
+        * transform categorical values #TODO
+* 2 - [Feature selection](#feature-selection) #WIP
+    * Select features using random forest classifier #WIP
+    
+* 3 - [Spot Check Algorithms](#spot-check)
+    * split dataset
+    * train on multiple algorithms  # WIP
+    
+
+## 2) plot_outlier_detection.ipynb
+
+This notebook contains a reference for some techniques to tackle the problem to detect and remove outliers sample from the data set.
```

Comment:
all throughout the text, `dataset` is one word, and not `data set` try to remove the spaces where required